### PR TITLE
Button Container and Styled Buttons - Dan

### PR DESCRIPTION
**Pull request by Danimal97**
Created a button container which uses the custom "ButtonInput" tag created in the presentational directory. The buttons are also styled and easily modifiable. Attached is an image showcasing 3 sample buttons.
Screen Shot 2019-03-26 at 3 04 58 PM

### CODE REVIEW

File: src/js/components/container/ButtonContainer.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+class ButtonContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      buttonPressed: 0
+    };
+    this.click = this.click.bind(this);
+  }
+  click(event){
```
Comment (bgeraci714) [inline src/js/components/container/ButtonContainer.js]: can you change the name to `onClickHandler`

File: src/js/components/container/ButtonContainer.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      this.setState({ [event.target.id]: event.target.value });
+  }
+  render() {
+    const { buttonPressed } = this.state;
+    return ( 
+        <div class = "btn-group">
+            <ButtonInput
+            label="buttonPressed"
```
Comment (bgeraci714) [inline src/js/components/container/ButtonContainer.js]: can you tab these over ;D

File: src/js/components/container/ButtonContainer.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            text="Clear"
+            type= {ButtonStyle}
+            id="buttonPressed"
+            value="Clear"
+            click={this.click}
+            />
+            <ButtonInput
+            label="buttonPressed"
```
Comment (bgeraci714) [inline src/js/components/container/ButtonContainer.js]: Can you add unique labels and ids.

File: src/js/components/presentational/ButtonInput.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      type={type}
+      className="button control"
+      id={id}
+      value={value}
+      onClick={click}
+      required
+    >
+    {text}
+    </ButtonStyle>
+    {//<label htmlFor={label}>{text}</label>
```
Comment (bgeraci714) [inline src/js/components/presentational/ButtonInput.js:16]: Can you remove unused code.

File: src/js/components/presentational/ButtonStyle.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+width: ${({width}) => (width ? `${width}px` : '50px')};
+height: ${({ height}) => (height ? `${height}px` : '25px')};
+color: ${({color}) => (color ? color:'blue')};
+cursor: ${({cursor}) => (cursor ? cursor: 'pointer')};
+name: ${({name}) => (name ? name:'Buttone')};
+background: ${({background}) => (background ? background:'lightblue')};
+label: ${({label}) => (label ? label:'inherit')};
```
Comment (bgeraci714) [inline src/js/components/presentational/ButtonStyle.js]: remove label, name, value, background => background-color, add a new line to the end, and as with Matt, lets switch from using pixels to 1-2 (or maybe even 3) rem for width/height.

File: webpack.config.js [hunk 6]
```diff
@@ -10,6 +10,11 @@ module.exports = {
           loader: "babel-loader"
         }
       },
+      {
+        test: /\.css$/,
```
Comment (bgeraci714) [inline webpack.config.js:14]: thank you, this is great!

**Review by bgeraci714 — CHANGES_REQUESTED** [review]
Looks great, couple small fixes please!

File: src/js/components/container/ButtonContainer.js (hunk 1 above)
Comment (Danimal97) [reply src/js/components/container/ButtonContainer.js]: Done!

File: src/js/components/container/ButtonContainer.js (hunk 2 above)
Comment (Danimal97) [reply src/js/components/container/ButtonContainer.js]: Done!

File: src/js/components/presentational/ButtonInput.js (hunk 4 above)
Comment (Danimal97) [reply src/js/components/presentational/ButtonInput.js:16]: Done!

File: src/js/components/presentational/ButtonStyle.js (hunk 5 above)
Comment (Danimal97) [reply src/js/components/presentational/ButtonStyle.js]: Done, with the exception of 'background', which is actually what it was supposed to be already.

File: webpack.config.js (hunk 6 above)
Comment (Danimal97) [reply webpack.config.js:14]: No problem!

File: src/js/components/container/ButtonContainer.js (hunk 3 above)
Comment (Danimal97) [reply src/js/components/container/ButtonContainer.js]: Done!

**Review by bgeraci714 — APPROVED** [review]

**Pull request merged**